Replacing the old concept of 'patch' by 'pull request' or 'contribution' (#18238)
django/django: docs/intro/contributing.txt

==============

For this tutorial, you'll need Git installed to download the current

> development version of Django and to generate a pull request for the changes you

Reviewer: This looks great but it needs to be wrapped at 79cols. Could you please adjust this sentence with that in mind?

Running Django's test suite for the second time
===============================================

> Once you've verified that your chnages and your test are working correctly, it's

Reviewer 2: Typo:
```suggestion
Once you've verified that your changes and your test are working correctly, it's
```

Author: @felixxm addressed in c525981b267f6fd7611d300e2df7666ca7d9ca4b

> Once you've verified that your changes and your test are working correctly, it's

Reviewer: Same here! :-)

Previewing your changes
=======================

> Now it's time to go through all the changes made in our contribution. To stage

Reviewer: I would suggest to use `branch` here...
```suggestion
Now it's time to go through all the changes made in our branch. To stage
```

More information for new contributors
-------------------------------------

> Before you get too into writing contributions for Django, there's a little more

Reviewer: Maybe?
```suggestion
Before you get too into contributing to Django, there's a little more
```

information on contributing that you should probably take a look at:

* You should make sure to read Django's documentation on
  :doc:`claiming tickets and submitting patches
  </internals/contributing/writing-code/submitting-patches>`.
  It covers Trac etiquette, how to claim tickets for yourself, expected

>   coding style for contributions, and many other important details.

Reviewer: I would just omit it here:
```suggestion
  coding style (both for code and docs), and many other important details.
```

------------------------------

Once you've looked through some of that information, you'll be ready to go out

> and find a ticket of your own to write a contribution for. Pay special attention

Reviewer: Maybe?
```suggestion
and find a ticket of your own to contribute to. Pay special attention
```

to tickets with the "easy pickings" criterion. These tickets are often much
simpler in nature and are great for first time contributors. Once you're

> familiar with contributing to Django, you can move on to writing contributions

Reviewer: ```suggestion
familiar with contributing to Django, you can move on to working on
```

After submitting a pull request, update the ticket metadata by setting the
flags on the ticket to say "has patch", "doesn't need tests", etc, so others
can find it for review. Contributing doesn't necessarily always mean writing a

> contribution from scratch. Reviewing existing contributions is also a very

Reviewer: ```suggestion
code from scratch. Reviewing existing contributions is also a very
```
